changelog.md

```diff
@@ -4,6 +4,7 @@
 
 - Updated `nodetime`: `ts-proto` to `v1.126.0`, `protobufjs` to `v7.1.1`, `swagger-typescript-api` to `v9.2.0`
 - Switched codegen client to use `axios` instead of `fetch`
+- nodetime built with `vercel/pkg@5.7.0`
```

Comment:
The 'Changes' section is missing in the changelog

```suggestion
### Changes

- Updated `nodetime`: `ts-proto` to `v1.126.0`, `protobufjs` to `v7.1.1`, `swagger-typescript-api` to `v9.2.0`
- Switched codegen client to use `axios` instead of `fetch`
- nodetime built with `vercel/pkg@5.7.0`
```